src/pages/projects/AddProjectManagerForm.tsx

```diff
@@ -56,11 +92,22 @@ export const AddProjectManagerForm = ({ toggleModal, suggestedProjectManager }:
         setSelectedPerson={setSelectedPerson}
         searchTerm={searchTerm}
         setSearchTerm={setSearchTerm}
-        selectAffiliations={SelectAffiliations.SINGLE}
+        selectAffiliations={hasAffiliation ? SelectAffiliations.NO_SELECT : SelectAffiliations.SINGLE}
       />
-      <StyledRightAlignedFooter>
+      <StyledRightAlignedFooter sx={{ mt: '2rem' }}>
+        {betaEnabled && (
```

Comment:
Dette er egentlig det `BetaFunctionality` er ment for, ved at man slipper å legge inn kode for å håndtere om det er beta eller ikke, i tillegg til at den gir en striplet linje rundt for å eksplisitt vise at det er betafunksjonalitet
```suggestion
        <BetaFunctionality>
```